backend/src/routes/api/status/index.ts

```diff
@@ -9,6 +9,7 @@ export default async (fastify: FastifyInstance): Promise<void> => {
         return res;
       })
       .catch((res) => {
+        fastify.log.error(res);
```

Comment:
Always add additional text before the log -- sometimes errors can be really unhelpful to showing where they came from.